pygeo/parameterization/DVGeoESP.py

```diff
@@ -640,7 +640,7 @@ def writeCADFile(self, filename):
         valid_filetypes = ["brep", "bstl", "egads", "egg", "iges", "igs", "sens", "step", "stl", "stp", "tess", "grid"]
         file_extension = filename.split(".")[-1]
         if file_extension.lower() not in valid_filetypes:
-            raise IOError(
+            raise OSError(
```

Comment:
Is `IOError` deprecated? Looks like [OSError](https://docs.python.org/3/library/exceptions.html#OSError) is more general

Reply:
IOError is indeed deprecated, from the link above:
```
Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and 
map.error have been merged into OSError, and the constructor may return a subclass.
```